charts/partials/templates/_proxy-init.tpl

```diff
@@ -1,5 +1,13 @@
 {{- define "partials.proxy-init" -}}
 args:
+{{- if (eq .Values.proxyInit.iptablesMode "nft") }}
+- --firewall-bin-path
+- {{"iptables-nft" | quote}}
```

Comment:
what does passing this through quote do vs just putting the quoted value directly in the template?

Reply:
I thought at first it stripped the quotes. I've gone through it again and it doesn't seem to be the case so I removed the `quote` fn call.